wireworld.py

```diff
@@ -0,0 +1,154 @@
+import json
+import sys
+import copy
+
+import numpy as np
+
+initial_state = [
+[0, 0, 0, 3, 0, 0, 0], 
+[0, 3, 0, 3, 0, 3, 0], 
+[3, 0, 3, 0, 3, 0, 3], 
+[1, 0, 0, 3, 0, 0, 1], 
+[2, 0, 3, 3, 3, 0, 2], 
+[0, 3, 0, 3, 0, 3, 0], 
+[0, 0, 0, 3, 0, 0, 0]
+]
+
+
+def config(initial_state):
+	print("To load a file, type in the filename. To use the default state, press enter")
+	filename = input()
+	if filename[-5:] != ".json":
```

Comment:
You can rejig these lines a bit, so that instead of checking `if filename != ""` twice, you can put that condinitonal on the upper level, and `filename[-5:]` inside of it.